Learn/controllers/HomeViewController.swift

```diff
@@ -19,44 +19,35 @@ class HomeViewController: UIViewController {
     @IBOutlet weak var velocityLabel: UILabel!
     @IBOutlet weak var nameLabel: UILabel!
     
-    
-    
-    
-    
-    
-    
-    
-    
     let store = UserDataStore.shared
 
-
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        store.fetchProgress { (studentWithProgress) in
-            print(studentWithProgress)
-            self.updateViewWith(studentWithProgress)
-            
+//        store.fetchProgress { (studentWithProgress) in
```

Comment:
Commented this out for now, want to discuss flow on this, but assuming we want to always go to lessons view after successful token check or login flow. For now though I opted into loading Profile as I was not seeing the lesson view flow in the current master code.